modules/services/hp-admin-crypto-server.nix

```
@@ -20,6 +20,8 @@ in
     systemd.services.hp-admin-crypto-server = {
       wantedBy = [ "multi-user.target" ];
 
+      environment.RUST_LOG = "debug";
```

```suggestion
      environment.RUST_LOG = "info";
```

`debug` is intended only for development purposes.

---

Related idea described in #371